Sources_v3/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -318,3 +343,7 @@ extension ClientError {
         }
     }
 }
+
+extension Int {
+    static let maxNumberOfLatestReactions = 10
+}
```

Comment:
This should ideally live in the config. Since we don't do that yet for these limits, just use the number directly in the code. This feels like a done solution and it's not.